### PR TITLE
feat: add create-package script to auto create packages

### PR DESCRIPTION
**Pull request by 0xnigir1**
## Description

A bash script that simplifies the scaffolding of a new package

## Checklist before requesting a review

- [x] I have conducted a self-review of my code.
- [x] I have conducted a QA.
- [x] If it is a core feature, I have included comprehensive tests.

### CODE REVIEW

**Review by 0xkenj1 — COMMENTED** [review]
- should we create another script for apps ?
- lets add instructions to run the scripts on the readme file

**Comment by 0xnigir1** [comment]
> * should we create another script for apps ?

i don't think is necessary for the apps, i don't expect that we have many apps and they don't necessarily share a common structure

> * lets add instructions to run the scripts on the readme file

Updated 🫡

File: create-package.sh [hunk 1]
```diff
@@ -0,0 +1,161 @@
+#!/bin/bash
+
+# Check if a package name is provided
+if [ $# -eq 0 ]; then
+    echo "Please provide a package name."
+    echo "Usage: ./create-package.sh <package-name>"
+    exit 1
+fi
+
+PACKAGE_NAME=$1
```
Comment (jahabeebs) [inline create-package.sh:10]: optional: could sanitize some invalid package names like
`
PACKAGE_NAME=$(echo "$1" | tr '[:upper:]' '[:lower:]' | tr -cs 'a-z0-9' '-')`

**Review by jahabeebs — APPROVED** [review]

File: create-package.sh (hunk 1 above)
Comment (0xnigir1) [reply create-package.sh:10]: what does it do?

**Review by 0xyaco — APPROVED** [review]
```
       _
     _|=|__________
    /   🔥🔥🔥🔥   \
   /     YOUR PR    \
  /__________________\
   ||  || /--\ ||  ||
   ||[]|| | .| ||[]||
 ()||__||_|__|_||__||()
( )|-|-|-|====|-|-|-|( )   🚒🚒🚒🚒
^^^^^^^^^^====^^^^^^^^^^^
```

File: create-package.sh (hunk 1 above)
Comment (jahabeebs) [reply create-package.sh:10]: converts to lowercase and replaces non-alphanumeric characters with -

"my_package123!" becomes "my-package123"

Comment (0xnigir1) [reply create-package.sh:10]: could be a nice improvement, we need a backlog for the boilerplate

**Pull request merged**